crates/sui-indexer-alt/Cargo.toml

```diff
@@ -42,6 +42,7 @@ sui-pg-temp-db.workspace = true
 sui-protocol-config.workspace = true
 sui-storage.workspace = true
 sui-types.workspace = true
+sui-synthetic-ingestion.workspace = true
```

Comment:
Seeing as we are making a fresh start -- is it worth making this an optional feature, guarded by a flag?